runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/status/ChannelzServlet.java

```diff
@@ -0,0 +1,245 @@
+package org.apache.beam.runners.dataflow.worker.status;
+
+import org.apache.beam.vendor.grpc.v1p60p1.io.grpc.channelz.v1.*;
+import org.apache.beam.vendor.grpc.v1p60p1.io.grpc.protobuf.services.ChannelzService;
+import org.apache.beam.vendor.grpc.v1p60p1.io.grpc.stub.StreamObserver;
+import org.apache.beam.vendor.guava.v32_1_2_jre.com.google.common.util.concurrent.SettableFuture;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/** Respond to /channelz with the GRPC channelz data. */
+@SuppressWarnings({
+        "nullness" // TODO(https://github.com/apache/beam/issues/20497)
```

Comment:
can this be fixed up to not require this?

Reply:
done